tracer.go

```diff
@@ -141,26 +141,26 @@ func (t *tracerImpl) startSpanInternal(
 	return sp
 }
 
-func (t *tracerImpl) Extractor(format interface{}) opentracing.Extractor {
+func (t *tracerImpl) Inject(toInject opentracing.Span, format interface{}, carrier interface{}) error {
 	switch format {
 	case opentracing.SplitText:
-		return t.textPropagator
+		return t.textPropagator.InjectSpan(toInject, carrier)
```

Comment:
I can rename InjectSpan/JoinTrace in the individual propagators if that's desired.